src/pointgroup.c

```diff
@@ -1121,8 +1121,8 @@ static void layer_check_and_sort_axes(int axes[3], int const aperiodic_axis) {
     } else {
         // Warning when rot_axes[axes[i]][aperiodic_axis] == -2, -3, 2, 3
         // I am not sure if this would ever happen.
```

Comment:
This code is introduced for layer group, but we will not want to expose the feature for users since we should reconsider the algorithm, and the implementation will be redesigned. This will be a work not only the code redesign but require some scientific discussion. The code lines for the layer group implementation were inserted in many places and the refactoring will need some effort and take time. So I just have been leaving it untouched.